<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>350</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/upButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed{
background-color: FF0000;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/downButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed{
background-color: FF0000;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/leftButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed{
background-color: FF0000;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/rightButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed{
background-color: FF0000;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/powerButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed{
background-color: FF0000;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/okButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed{
background-color: FF0000;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="deviceBackground">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>69</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chargeBattery">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>550</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Charge Battery</string>
    </property>
   </widget>
   <widget class="QPushButton" name="coherenceLightRed">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(191, 64, 64);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="coherenceLightBlue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 74, 135);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="coherenceLightGreen">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="session">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Session Underway...</string>
    </property>
   </widget>
   <widget class="QFrame" name="Summary">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="SumLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Summary of your session</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SummaryButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>310</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="AverageCoherence">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>250</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Average Coherence:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SessionLength">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>280</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Length of Session:</string>
     </property>
    </widget>
    <widget class="QLabel" name="AchievementScore">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>310</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Achievement Score:</string>
     </property>
    </widget>
    <widget class="QCustomPlot" name="summaryGraph" native="true">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>271</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lowCoherence">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>LC (%):</string>
     </property>
    </widget>
    <widget class="QLabel" name="mediumCoherence">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>290</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MC (%):</string>
     </property>
    </widget>
    <widget class="QLabel" name="highCoherence">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>310</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>HC (%):</string>
     </property>
    </widget>
    <widget class="QLabel" name="AverageCoherence_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Coherence Time Percentage (approx.)</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Settings">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="settingsLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="breathPacerLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Breath Pacer</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="breathPacer">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
    </widget>
    <widget class="QPushButton" name="setSettings">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="skinLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply to Skin</string>
     </property>
    </widget>
    <widget class="QComboBox" name="applyToSkin">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>72</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="highButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>High</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mediumButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Medium</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lowButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Low</string>
     </property>
    </widget>
    <widget class="QLabel" name="skinLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Coherence LVL</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>331</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 87, 83);
border-radius: 20px;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(239, 41, 41);
border-color: rgb(0, 0, 0);
font: 12pt &quot;Noto Sans&quot;;</string>
     </property>
     <property name="text">
      <string>HeartWave</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>331</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69, 69, 69);
border-color: rgb(0, 0, 0);
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="batteryLevel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>80</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;100&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QListWidget" name="menuList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>191</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom-right-radius: 12px;
border-bottom-left-radius: 12px;
background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
selection-background-color: rgb(122, 0, 0)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="breath">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>widget</zorder>
   <zorder>deviceBackground</zorder>
   <zorder>leftButton</zorder>
   <zorder>rightButton</zorder>
   <zorder>okButton</zorder>
   <zorder>upButton</zorder>
   <zorder>powerButton</zorder>
   <zorder>downButton</zorder>
   <zorder>chargeBattery</zorder>
   <zorder>coherenceLightRed</zorder>
   <zorder>coherenceLightBlue</zorder>
   <zorder>coherenceLightGreen</zorder>
   <zorder>session</zorder>
   <zorder>Summary</zorder>
   <zorder>Settings</zorder>
   <zorder>batteryLevel</zorder>
   <zorder>customPlot</zorder>
   <zorder>breath</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
